.gitignore

```diff
@@ -99,3 +99,4 @@ ENV/
 
```

Comment:
**Tip:** _I don't know if this applies as good practice_
Consider removing the .gitignore file from git tracker because it only applies to your computer. If I clone your repo, the ignore file might not work with my local setup.